docs/features/software-catalog/descriptor-format.md

```diff
@@ -700,6 +700,11 @@ sufficient to enter only the `metadata.name` field of that group.
 
 ### `spec.ancestors` [required]
 
+**NOTE**: This field was marked for deprecation on Nov 22nd, 2020. It will be
```

Comment:
The day of the release that introduces the deprecation would be the 26th, so removal would be the 10th

Reply:
This is related to the comment above from me - since the docs aren't versioned, and some users use master and some use released libs, it wasn't entirely clear which set of dates that made the most sense to list here. Considered listing all of the involved dates (including the first release after marking deprecation, and the first release after deletion), but it felt like it almost made it harder to understand.